lib/reek/context_builder.rb

```diff
@@ -74,7 +77,11 @@ def process_module(exp)
       end
     end
 
-    alias_method :process_class, :process_module
+    def process_class(exp)
+      inside_new_context(Context::ClassContext, exp) do
```

Comment:
I was on the make-all-a-context bandwagon but now on second thought after your comment I realized that it doesn't improve our architecture by just adding this by default. I restore the initial

>> alias_method :process_class, :process_module